app/src/main/java/com/gmail/nowak/wjw/popularmovies/presenter/list/MovieListActivity.java

```diff
@@ -0,0 +1,310 @@
+package com.gmail.nowak.wjw.popularmovies.presenter.list;
+
+import android.content.Intent;
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.databinding.DataBindingUtil;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProviders;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.gmail.nowak.wjw.popularmovies.data.model.view_data.list.MovieListItemViewData;
+import com.gmail.nowak.wjw.popularmovies.presenter.detail.DetailActivity;
+import com.gmail.nowak.wjw.popularmovies.R;
+import com.gmail.nowak.wjw.popularmovies.data.model.local.FavouriteMovie;
+import com.gmail.nowak.wjw.popularmovies.databinding.ActivityMainBinding;
+import com.gmail.nowak.wjw.popularmovies.utils.NetworkUtils;
+
+import java.util.List;
+
+import timber.log.Timber;
+
+public class MovieListActivity extends AppCompatActivity implements MovieAdapter.OnMovieListItemClickListener {
+
+    //todo in stage 3:
+    // - implement fragments holding each list - dissect ListActivity to three fragments
+    // - replace DetailActivity with DetailFragment - make it one activity app
+    // - implement dagger
+    // - implement javaRx
+    // - enable watching videos inside the app
+
+    private static final String POPULARITY_TAG_TITLE = NetworkUtils.POPULARITY_TAG_TITLE;
+    private static final String TOP_RATED_TAG_TITLE = NetworkUtils.TOP_RATED_TAG_TITLE;
+    private static final String FAVOURITE_TAG_TITLE = "Favourite";
+    //todo Q? should I use Enum to make it accessible for other classes and stored in one place?  how about POPULARITY_TAG_TITLE?
+    private static final int POPULAR_MOVIES_TAG = 0;
+    private static final int TOP_RATED_MOVIES_TAG = 1;
+    private static final int FAVOURITE_MOVIES_TAG = 2;
+    public static final String DISPLAYED_LIST_TAG = "displayed_list";
+
+    public static final String EXTRA_API_ID = "extra_api_id";
+    private MovieAdapter movieAdapter;
+    private ActivityMainBinding binding;
+
+    MovieListViewModel viewModel;
+    // stores the currently displayed tab's tag
+    private int displayedTab = POPULAR_MOVIES_TAG;
```

Comment:
Make enum from displayTab 

Advanced tip: use private constructors (with params) in those enums to store tab title.

 enum DisplayedTab {
        TOP("a"),
        FAV("b");

        private String title;
        private DisplayedTab(String title) {
            this.title = title;
        }
    }
//declaration:
private DisplayedTab displayedTabb;

// change selectoin:
displayedTabb = DisplayedTab.TOP;
        
//change title:
 binding.   ... = displayedTabb.title